packages/backend-ai/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@aws-amplify/backend-ai",
+  "version": "0.1.0",
+  "type": "module",
+  "publishConfig": {
+    "access": "public"
+  },
+  "exports": {
+    ".": {
+      "types": "./lib/index.d.ts",
+      "import": "./lib/index.js",
+      "require": "./lib/index.js"
+    }
+  },
+  "dependencies": {
+    "@aws-sdk/client-bedrock-runtime": "^3.602.0",
+    "aws-cdk-lib": "^2.132.0",
+    "aws-sdk": "^2.1652.0"
+  },
+  "peerDependencies": {
+    "aws-cdk-lib": "^2.132.0",
+    "constructs": "^10.0.0"
+  },
+  "types": "lib/index.d.ts",
+  "scripts": {
+    "update:api": "api-extractor run --local"
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "aws-sdk-mock": "^6.0.4"
```

Comment:
we don't need this package. Just use `mock.method` from `node:test` to mock SDK client methods